tests/plugins/org.polarsys.capella.test.diagram.filters.ju/src/org/polarsys/capella/test/diagram/filters/ju/lab/HideElementTitleBlocksTest.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2020 THALES GLOBAL SERVICES.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *  
+ * Contributors:
+ *    Thales - initial API and implementation
+ *******************************************************************************/
+package org.polarsys.capella.test.diagram.filters.ju.lab;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.polarsys.capella.core.sirius.analysis.constants.IFilterNameConstants;
+import org.polarsys.capella.test.diagram.filters.ju.DiagramTitleBlockFilterTestCase;
+
+public class HideElementTitleBlocksTest extends DiagramTitleBlockFilterTestCase {
+
+  private final String ELEMENT_TITLE_BLOCK_LC1_ID = "_P6x4cI4QEeqaDKEiylAJ0A";
+  private final String ELEMENT_TITLE_BLOCK_LC2_ID = "_YH2OQI4VEeqaDKEiylAJ0A";
+

```

Comment:
Also you could add a function in the model and put a TB on a function (to test that on multiple types of components, functions etc).